charts/kof-storage/templates/jaeger/ingress.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.jaeger.enabled }}
+{{- if .Values.jaeger.ingress.enabled}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: jaeger-query-ingress
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    kubernetes.io/tls-acme: "true"
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: {{ .Values.jaeger.ingress.queryHost | quote }}
+    http:
+      paths:
+      - backend:
+          service:
+            name: {{ .Release.Name }}-jaeger-query
+            port:
+              number: 16686
+        path: /
+        pathType: Prefix
+  tls:
+  - secretName: jaeger-query-tls-secret
+    hosts:
+      - {{ .Values.jaeger.ingress.queryHost | quote }}
+  
+---
+apiVersion: networking.k8s.io/v1
```

Comment:
Probably you don't need two ingresses here, as you can route different paths to different backends under paths section. Have you tried that?